pkgs/development/tools/build-managers/apache-maven/builder.sh

```diff
@@ -5,7 +5,7 @@ unpackPhase
 mkdir -p $out/maven
 cp -r $name/* $out/maven
 
-makeWrapper $out/maven/bin/mvn $out/bin/mvn --set JAVA_HOME "$jdk"
+makeWrapper $out/maven/bin/mvn $out/bin/mvn --set JAVA_HOME "$jdk/lib/openjdk"
```

Comment:
Oooooh, good catch. Let me see what I can do about that. It may actually then be easier to create a symlink for the folder in each JDK rather than fixing it here. (Similarly to what we seem to have for bin/, which symlinks further down)